pychunkedgraph/app/segmentation/v1/routes.py

```diff
@@ -121,6 +121,17 @@ def handle_root(table_id, node_id):
     return jsonify_with_kwargs({"root_id": root_id}, int64_as_str=int64_as_str)
 
 
+### GET ROOTS -------------------------------------------------------------------
+
+
+@bp.route("/table/<table_id>/roots", methods=["POST"])
```

Comment:
Deployed it and it is not working for me. I believe this needs to be "GET"

Reply:
Adding request bodies to a `GET` request is possible, but more confusing / less expected than using a `POST` request, even if we are not updating any resources here.

A decade old question: https://stackoverflow.com/questions/978061/http-get-with-request-body